lib/Runtime/Base/FunctionBody.cpp

```diff
@@ -4373,6 +4372,13 @@ namespace Js
 /* END potentially reusable code */
 #endif /* IR_VIEWER */
 
+#if ENABLE_TTD
+    void FunctionBody::GetSourceLineFromStartOffset_TTD(const uint startOffset, ULONG* line, LONG* col)
```

Comment:
We may want to adjust this API to a faster implementation in the future so I wanted to keep the uses separate.